tests/test_filter_panels.py

```diff
@@ -0,0 +1,63 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.treeherder import TreeherderPage
+
+
+@pytest.mark.nondestructive
+def test_edit_filter_panels(base_url, selenium):
+    """Open Treeherder page, open Filters Panel, select one filter,
+    verify results"""
+    page = TreeherderPage(selenium, base_url).open()
+    assert page.unclassified_failure_count > 0
+
+    # Get next 50 jobs to see 'exception' unclassified failures
+    page.get_next_fifty_results()
```

Comment:
True. So I will delete this test. Should I split rest of the tests cases (testfailed, busted) for different functions?